packages/sitebuilder/src/utils/parseDomainListItem.ts

```diff
@@ -10,13 +10,24 @@ type DomainListItem = {
 	selected: boolean,
 }
 
-function getPrice(price: string, isAvailable: boolean) {
+function getPrice(termFees: DomainTermFees, isAvailable: boolean) {
```

Comment:
As the function is now a bit more complicated, I think it is a good idea to unit-test it.